snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/SnomedConceptConverter.java

```diff
@@ -119,14 +121,12 @@ protected SnomedConcept toResource(final SnomedConceptIndexEntry input) {
 	
 	@Override
 	protected void expand(List<ISnomedConcept> results) {
-		final Set<String> conceptIds = FluentIterable.from(results).transform(ID_FUNCTION).toSet();
-		expandInactivationProperties(results, conceptIds);
-		
-		if (expand().isEmpty()) {
+		if (expand().isEmpty() || results.isEmpty()) {
 			return;
 		}
 		
-		
+		final Set<String> conceptIds = FluentIterable.from(results).transform(ID_FUNCTION).toSet();
+		expandInactivationProperties(results, conceptIds);
```

Comment:
On 5.x we put this behind an expand key: https://github.com/b2ihealthcare/snow-owl/blob/eac279977bf2b0c49da949701136ba19260b5aef/snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/SnomedConceptConverter.java#L188

Is it possible to do so here as well?